WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2054,3 +2054,19 @@ ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
 MetaPropertyTable       =
 MetaPropertyTableCell   = padding:@SkinWrapperPadding@ @WidgetInternalPadding@; padding-top:0;
 
+#######################################################################
+#   Task List view
+#######################################################################
+
+TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right:12px;
+TaskListViewNewTaskBannerWrapper                            = @Text-hint@ border-bottom:1px solid @SeparatorColor@; @InactiveCursor@ @FontSize-big@ padding:16px 0 16px 6px;
+TaskListViewHeaderItemLabel                                 = @FontSize-slightly-big@ @Text-light@ padding: 10px 0px 10px 11px;
+TaskReadingPaneRight-ListViewItemCheckboxCell               = padding-left:10px;
+TaskReadingPaneRight-ListViewItem                           = height:80px;
+TaskReadingPaneRight-ListViewItemSubject                    = vertical-align:bottom;
+TaskReadingPaneRight-ListViewItemDate                       = align:right; padding-top:16px;
```

Comment:
align:right property was used in existing code and I moved it to class level.
This is a td and align works on that.
Plz refer ::--
https://www.w3schools.com/tags/att_td_align.asp
Using vertical-align:middle, will move the item little bit up then what is required.
So avoided using that.